Safety/Src/PWM.cpp

```diff
@@ -36,6 +33,13 @@ PWMChannel::PWMChannel()
     {
         PWMPinConfig currentChannel = PWM_CONFIG[i];
         HAL_TIM_PWM_Start(currentChannel.timer,currentChannel.timer_channel);
+
+        if (currentChannel.isUsingDshot)
+        {
+                //setting up dma call back function that runs when the transfer is complete and disables the dma
+                dshotSetupDMACallbacks();
+        }
```

Comment:
Wait hmmm shouldn't we be getting a pointer to what's in PWM_Config rather than an object? doesn't it create a copy of the object this way? I suppose it has the same effect in the end, but it would be better to do PWMPinConfig *currentChannel = &PWM_Config[i]

(This is not your fault LOL, in fact it might be mine, but I recommend editing it in this PR)

Reply:
This leads to an error of "error: invalid conversion from 'const PWMPinConfig*' to 'PWMPinConfig*'"

Reply:
It seems to me like you would just need to make `PWMPinConfig currentChannel` a const variable to fix this. eg. `const PWMPinConfig *currentChannel = &PWM_CONFIG[i];`